src/V3AstNodeDType.h

```diff
@@ -61,9 +61,9 @@ class AstNodeDType VL_NOT_FINAL : public AstNode {
     /// under another compound-requiring object e.g. class
     virtual bool isCompound() const = 0;
     // (Slow) recurse down to find basic data type
-    virtual AstBasicDType* basicp() const = 0;
+    virtual AstBasicDType* basicp() const VL_MT_SAFE = 0;
     // recurses over typedefs/const/enum to next non-typeref type
-    virtual AstNodeDType* skipRefp() const = 0;
+    virtual AstNodeDType* skipRefp() const VL_MT_SAFE = 0;
```

Comment:
If you're going to set VL_MT_SAFE in a base class, all the overridden functions need to likewise have the VL_MT_SAFE annotation.